main.go

```diff
@@ -74,15 +74,37 @@ func handleIssueComment(body []byte, gitRepos git.Repos, pullRequests PullReques
 	if !issueComment.IsPullRequest {
 		return SuccessResponse{"Not a PR. Ignoring."}
 	}
-	switch {
-	case isSquashCommand(issueComment.Comment):
+	commentCategory := parseComment(issueComment.Comment)
+	if commentCategory == regularComment {
+		return SuccessResponse{"Not a command I understand. Ignoring."}
+	}
+	if isAuthorized, err := isCollaborator(issueComment.Repository, issueComment.User, repositories); err != nil {
```

Comment:
I was thinking about doing that, but the issue here is that there's 3 possible ways I might want to continue after calling that utility function: 1) I might want to return an error response because something went wrong, 2) I might want to return early with a success response, because the user is not authorized and we let them know so, or 3) I might want to continue. I couldn't initially think of a good way of doing that, but did right now: https://github.com/salemove/github-review-helper/pull/25/commits/8d465d28788d6b05a49af08a04a5efa0b67de6fc